src/buildConfig.js

```diff
@@ -106,6 +111,18 @@ function getModulePath(basedir, lookup) {
   )
 }
 
+function findIgnorePatterns(configDir) {
+  const ignoreFilePath = path.resolve(configDir, ".stylelintignore")
```

Comment:
Since the name of this file is a true constant, let's set `const IGNORE_FILENAME = ".stylelintignore"` at the very top of this file.